controller_manager/doc/userdoc.rst

```diff
@@ -8,11 +8,29 @@ It manages lifecycle of controllers, access to the hardware interfaces and offer
 Determinism
 -----------
 
-For best performance when controlling hardware you want the controller manager to have as little jitter as possible in the main control loop.
-The normal linux kernel is optimized for computational throughput and therefore is not well suited for hardware control.
-The main thread of Controller Manager attempts to configure ``SCHED_FIFO`` with a priority of ``50``.
-To enable this functionality install a RT kernel and run the Controller Manager with permissions to make syscalls to set its thread priorities.
-The two easiest options for this are using the [Real-time Ubuntu 22.04 LTS Beta](https://ubuntu.com/blog/real-time-ubuntu-released) or [linux-image-rt-amd64](https://packages.debian.org/bullseye/linux-image-rt-amd64) on Debian Bullseye.
+For best performance when controlling hardware you want the controller manager to have as little jitter as possible in the main control loop. The normal linux kernel is optimized for computational throughput and therefore is not well suited for hardware control. The two easiest kernel options are the [Real-time Ubuntu 22.04 LTS Beta](https://ubuntu.com/blog/real-time-ubuntu-released) or [linux-image-rt-amd64](https://packages.debian.org/bullseye/linux-image-rt-amd64) on Debian Bullseye.
```

Comment:
Each sentence should be on its own line. Because of how rst works they will be collected into a paragraph if you do this and it will render correctly while making diffs easier to read.

Reply:
![text](https://user-images.githubusercontent.com/11284393/179763286-b47d89dc-40c0-47e2-8850-d7d599e77b6d.png)

Reply:
You're mixing Markdown syntax for links with reStructuredText.

Reply:
Ah, OK. That was actually @tylerjw previously. Thanks, will fix

Reply:
That was my mistake. You need to use the rest link syntax. But also you should leave these on separate lines.

Reply:
@AndyZe: it's called using *semantic newlines*.

Reply:
![txt](https://user-images.githubusercontent.com/11284393/179765661-fb7a21e2-2e1d-40eb-a384-d43a8056f2ae.png)